### DISCUSSION TITLE
Tracking Repos Where You're A Collaborator

### DISCUSSION THREAD

**Discussion by gfleetwood** [community/community#28163] [Other Feature Feedback, Questions, & Ideas]
I was recently added to an org's repo as a collaborator and I can't seem to find any trace of it on GitHub -  had to use the cli. Is there someplace to see repos you're a collaborator on either for an org or another user?

**Comment by jfullstackdev** [comment]
do you know the name of the repo ? then search it at your scope and once you are there, bookmark it

**Comment by gfleetwood** [comment]
I found the url through the cli and starred the repo. It would just be useful for that to be a GitHub feature, i.e being able to see a list of repos you've been added to as a collaborator just like you can see which organizations you're a part of.

**Reply by ZajithCorro** [reply]
If you are part of the organization, you can switch on the dashboard between your profile and the organization. Maybe, this can help you yo track some repos 

![image](https://user-images.githubusercontent.com/23388616/182634448-602b2529-d9cf-43d4-a9dd-e043640caf90.png)

**Comment by jo-fleck** [comment]
This question comes up a lot and there is apparently no convenient fix so far:
https://stackoverflow.com/questions/12453358/see-repositories-where-i-am-collaborator

Does anyone know how to ask for/submit a request for this feature?

**Comment by jfullstackdev** [comment]
there is already the solution, when you are at your dashboard, it will appear, just like this 

![image](https://user-images.githubusercontent.com/47092464/194704085-bbb185cd-ed03-4674-9dc7-3b41a1123bc3.png)

it will not appear on your Repo tab
